src/components/form/FormFieldSliderNumber.vue

```diff
@@ -53,13 +56,29 @@ export default defineComponent({
     },
   },
   setup(props, { emit }) {
+    // compute default min from store
+    const challengeStore = useChallengeStore();
+    const defaultMin = computed(() => {
+      const currentPriceLevels = challengeStore.getCurrentPriceLevels;
```

Comment:
Same as my comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/814/files#r1899911173.

Reply:
Fixed with commit https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/814/commits/7671dbde12c708b595cf8e9ceff791533fa5bd0e.